docs/user/introduction.asciidoc

```diff
@@ -144,13 +145,10 @@ You might be managing tens, hundreds, or even thousands of dashboards, visualiza
 ==== Collect related items in a space
 
 {kib}
```

Comment:
Since spaces are unique to Kibana, could this just say, `Spaces are build for organizing...`?